components/AreaChartRenderer/components/range-toggle/RangeToggle.tsx

```diff
@@ -33,23 +35,48 @@ const TOGGLE_SET = [
 interface RangeToggleProps {
   onClick: (type: Range) => void;
   activeRange: string;
+  className?: string;
 }
 
-const RangeToggle: FC<RangeToggleProps> = ({ onClick, activeRange }) => (
-  <div className={styles.root}>
-    {TOGGLE_SET.map(({ label, type }) => (
-      <Button
-        key={type}
-        size="small"
-        className={cn(styles.toggle, {
-          [styles.active]: activeRange === type,
-        })}
-        onClick={() => onClick(type)}
-      >
-        {label}
-      </Button>
-    ))}
-  </div>
-);
+const RangeToggle: FC<RangeToggleProps> = ({
+  onClick,
+  activeRange,
+  className,
+}) => {
+  const isMobile = useMedia('(max-width: 768px)');
```

Comment:
I would suggest to astro mobile 920 px. What do you think?)

Reply:
fixed